docs/content/configuration/csi_ug_config_create_vol_replicationclass.md

```diff
@@ -0,0 +1,29 @@
+# Creating a VolumeReplicationClass
+
+Create a VolumeReplicationClass YAML file to enable volume replication.
+
+**Note:** Replication is also known as mirroring in many storage systems.
+
+In order to enable volume replication for your storage system, create a VolumeReplicationClass YAML file, similar to the following demo-replicationclass.yaml.
+
+When configuring the file, be sure to use the same array secret and array secret namespace as defined in [Creating a Secret](csi_ug_config_create_secret.md).
+
+-   The `snapshot_name_prefix` parameter is optional.
```

Comment:
The `snapshot_name_prefix` parameter is not relevant for replication, we did not add the replication yamls to our examples on GitHub yet, but feel free to sync with the yamls in our wiki.
the missing parameters here are `system_id` and `copy_type`.
how to find the `system_id` might require a separate page